en/docs/ai-gateway/next/about-this-release.md

```
@@ -0,0 +1,119 @@
+# AI Gateway Changelog
+
+**Release date:** 2026-07-07  
+**Previous version:** 1.1.0 (2026-04-30)  
+
+### New feature additions
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)